### PR TITLE
Feature/vmo 6586/show hide icon for block editor

### PR DESCRIPTION
**Pull request by bulatgab**
Demo:
Screen Shot 2022-09-02 at 4 49 53 PM
Screen Shot 2022-09-02 at 4 50 04 PM

### CODE REVIEW

File: src/assets/messages.json [hunk 1]
```diff
@@ -1606,7 +1607,6 @@
     "view-versions-issue-trackers": "View Versions & Issue Trackers",
     "view-x-other-versions": "View :count other versions of this tree",
     "voice": "Voice",
-    "voice-key-press": "Voice key press",
```
Comment (bulatgab) [inline src/assets/messages.json:1609]: duplicate

File: src/components/interaction-designer/blocks/BlockToolbar.vue [hunk 2]
```diff
@@ -7,13 +7,13 @@
           v-if="isBlockSelected"
           v-b-tooltip.hover="trans('flow-builder.deselect-block')"
           :icon="['far', 'check-circle']"
-          class="fa-btn text-info"
+          class="cursor-pointer text-info"
```
Comment (bulatgab) [inline src/components/interaction-designer/blocks/BlockToolbar.vue:10]: `fa-btn` wasn't applied for some reason.
Can someone please check and post below where that css class is located?

File: src/components/interaction-designer/blocks/BlockToolbar.vue [hunk 3]
```diff
@@ -43,9 +43,9 @@
           v-b-tooltip.hover="trans('flow-builder.tooltip-delete-block')"
           :icon="['far', 'trash-alt']"
           :class="{
-              'text-danger': !isWaitingForConnection && !isActivatedByConnection,
-            }"
```
Comment (bulatgab) [inline src/components/interaction-designer/blocks/BlockToolbar.vue:47]: eslint indentation fix

File: src/components/interaction-designer/blocks/BlockToolbar.vue [hunk 4]
```diff
@@ -111,7 +113,7 @@ export class BlockToolbar extends mixins(Lang) {
 
   handleExpandMinimizeBlockEditor(): void {
     this.setIsBlockEditorOpen(!this.isEditorVisible)
-    let routerName = ''
+    let routerName
```
Comment (bulatgab) [inline src/components/interaction-designer/blocks/BlockToolbar.vue:116]: a fix suggested by PhpStorm's code analysis

File: src/assets/messages.json [hunk 5]
```diff
@@ -3281,12 +3282,11 @@
     "view-versions-issue-trackers": "Voir les versions et les suivis de problèmes",
     "view-x-other-versions": "Voir :count autres versions de cet arbre",
     "voice": "Voix",
-    "voice-key-press": "Appui vocal sur la touche",
     "voice-choice-mapping-expression": "Écrivez une expression pour vérifier si la réponse des touches du contact (stockée dans block.response ) correspond à ce choix.",
     "voice-content": "Contenu Voix",
     "voice-disabled": "Voix désactivée",
     "voice-enabled": "Voix activée",
-    "voice-key-press": "Touche pour la voix",
+    "voice-key-press": "Appui vocal sur la touche",
```
Comment (bulatgab) [inline src/assets/messages.json:3289]: This is the latest localization version by Safydy

File: src/components/interaction-designer/blocks/BlockToolbar.vue (hunk 2 above)
Comment (object-required) [reply src/components/interaction-designer/blocks/BlockToolbar.vue:10]: I couldn't find `fa-btn` either. It appears it should come from Font Awesome, but there are no references there.

**Review by object-required — APPROVED** [review]

File: src/components/interaction-designer/blocks/BlockToolbar.vue (hunk 2 above)
Comment (bulatgab) [reply src/components/interaction-designer/blocks/BlockToolbar.vue:10]: Thanks!

**Pull request merged**